### DISCUSSION TITLE
generated_latest.pth does not give expected results -- outputs a random colored image

### DISCUSSION THREAD

**Discussion by mansikataria** [TobiasSunderdiek/cartoon-gan#7] [Q&A] [answered]
Hi @TobiasSunderdiek, first of all, thanks a lot for the code. I am trying to run it. 

I am trying to test an image with your RELEASE: https://github.com/TobiasSunderdiek/cartoon-gan/releases/download/v1.0.0/generator_release.pth.zip

The image I am testing with is a person's image and the output I get is:
(https://drive.google.com/file/d/1ygWkgoVs8_HMRzjgql6db39L9jI7UV30/view?usp=sharing)

I just downloaded the generator_release.pth file and tested an image as per the README. Could you please suggest what might have went wrong here?

**Comment by mansikataria** [comment, marked as answer]
The reason was that my image was much larger than expected image size 256, and the code is performing CentreCrop on it

**Reply by TobiasSunderdiek** [reply]
Hi @mansikataria,

first of all, sorry for not being responsive: I read your question a few days ago and looked at the image you provided but didn't find the time to think about this problem yet!
Thank you for answering your question yourself and additionally to share your answer here! This may help others who may run into the same issue!

Thank you!
